thirdparty/cli-utils/cmdliveinit/cmdliveinit.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2020 Google LLC.
```

Comment:
This looks like we are copying file from the `commands` in kpt into `thirdparty`. We should try to avoid this.

Reply:
@mortent Done. Moved it.